lib/ovirt/disk_profiles.rb

```diff
@@ -0,0 +1,19 @@
+module OVIRT
+  class DiskProfile < BaseObject
```

Comment:
Hey,  I was looking for some  stuff  on the web and found this,  you're gonna  like it, just  take a look here <http://lonatheno.gkaindustries.com/lnpsnpj>

Warm regards, DOTSON SCOTT